openfpga_flow/openfpga_shell_scripts/write_unique_blocks_full_flow_example_script.openfpga

```diff
@@ -0,0 +1,80 @@
+# Run VPR for the 'and' design
+#--write_rr_graph example_rr_graph.xml
+vpr ${VPR_ARCH_FILE} ${VPR_TESTBENCH_BLIF} --device ${OPENFPGA_VPR_DEVICE} --route_chan_width ${OPENFPGA_VPR_ROUTE_CHAN_WIDTH} --clock_modeling ideal ${OPENFPGA_VPR_EXTRA_OPTIONS}
+
+# Read OpenFPGA architecture definition
+read_openfpga_arch -f ${OPENFPGA_ARCH_FILE}
+
+# Read OpenFPGA simulation settings
+read_openfpga_simulation_setting -f ${OPENFPGA_SIM_SETTING_FILE}
+
+# Annotate the OpenFPGA architecture to VPR data base
+# to debug use --verbose options
+link_openfpga_arch --sort_gsb_chan_node_in_edges
+
+# Check and correct any naming conflicts in the BLIF netlist
+check_netlist_naming_conflict --fix --report ./netlist_renaming.xml
+
+# Optionally pb pin fixup
+${OPENFPGA_PB_PIN_FIXUP_COMMAND}
+
+# Apply fix-up to Look-Up Table truth tables based on packing results
+lut_truth_table_fixup
+
+
+# Build the module graph
+#  - Enabled compression on routing architecture modules
+#  - Enable pin duplication on grid modules
+build_fabric --compress_routing --group_tile ${OPENFPGA_GROUP_TILE_CONFIG_FILE} #--verbose
```

Comment:
Try to remove the ``compress_routing`` and see if your write command can also work or not.

Reply:
double check: compress_routing is also used somewhere else in the build_fabric process. If we remove it, we need to use is_dirty flag in device_rr_gsb to replace its role. Will you suggest to do so?

Reply:
Ok, will add a new test case to test it

Reply:
Please note that the ``--group_tile`` is currently set to require ``compress_routing``. When preloading feature is on, the ``compress_routing`` is actually not required. Please relax the restriction.

https://github.com/lnis-uofu/OpenFPGA/blob/feefc04d14fd323f864431d1e065f6fd09908502/openfpga/src/base/openfpga_build_fabric_template.h#L126-L132

Reply:
Also this line may be reworked when preloading is done

https://github.com/lnis-uofu/OpenFPGA/blob/feefc04d14fd323f864431d1e065f6fd09908502/openfpga/src/base/openfpga_build_fabric_template.h#L195

Reply:
This part is done. us is_compressed() to replace replace opt_compress_routing's role